bundle/metadata/annotations.yaml

```diff
@@ -13,5 +13,3 @@ annotations:
   # Annotations for testing.
   operators.operatorframework.io.test.mediatype.v1: scorecard+v1
   operators.operatorframework.io.test.config.v1: tests/scorecard/
-
-  com.redhat.openshift.versions: "v4.8"
```

Comment:
I think we need this for openshit release, no?

Reply:
Yeah, I think it makes sense to bring it back 👍